router_test.go

```diff
@@ -178,6 +179,47 @@ func TestRouterDefault(t *testing.T) {
 	_ = NewTestRouter(t)
 }
 
+func TestRouterConfigurableCors(t *testing.T) {
+	cfg := cors.DefaultConfig()
+	cfg.AllowAllOrigins = true
+	cfg.AllowHeaders = append(cfg.AllowHeaders, "Authorization", "X-Istreamplanet-User-Identity")
```

Comment:
Can we use a different example so we don't leak iSP-specific stuff?

Reply:
At 3dc764f